content/concepts/introduction/users.md

```diff
@@ -13,43 +13,42 @@ A few notable users of libp2p include (3 most notable users listed first with th
+| **[Nethermind](https://www.nethermind.io/)** | By leveraging libp2p, Nethermind has positioned its projects (Juno and dotnet-libp2p) to contribute significantly to the decentralization and robustness of the Ethereum ecosystem. |
 | **[Nillion](https://nillion.com/)** | Compute nodes in the Nillion network are configured as relay servers using libp2p's Circuit Relay protocol. |
 | **[Nym](https://nym.com/)** | Nym's mixnet technology is integrated with libp2p to provide network-level privacy protection. The Nym libp2p module can be used by Ethereum validators to shield their network traffic from surveillance. |
 | **[Oasis Protocol](https://oasisprotocol.org/)** | Oasis utilizes libp2p for peer-to-peer communication in its consensus layer, which is responsible for maintaining the network's state and validating transactions. |
-| **[Optimism](https://optimism.io/)** | Optimism mainnet uses libp2p as a core component of its networking infrastructure, particularly in its op-node implementation. |
+| **[Optimism](https://optimism.io/)** | Optimism mainnet uses libp2p as a core component of its networking infrastructure, particularly in its op-node implementation. Furthermore, the Optimism Collective have developed the standardized, shared, and open-source development stack called [OP Stack](https://docs.optimism.io/stack/getting-started) which enables the creation of new L2 blockchains that rely on libp2p.|
 | **[Peergos](https://peergos.org/)** | Peergos uses libp2p to build a peer-to-peer encrypted global filesystem with fine-grained access control, enhancing user privacy and data security. |
-| **Polkadot** | Polkadot leverages libp2p as part of its Substrate-based architecture. Libp2p enables Polkadot's scalability by facilitating seamless communication across its heterogeneous blockchain network. |
-| **[Prysm (Prysmatic Labs)](https://prysmaticlabs.com/)** | The adoption of libp2p has allowed Prysm to be interoperable with other Ethereum consensus clients, contributing to client diversity in the Ethereum ecosystem. |
+| **[Polkadot](https://polkadot.com/)** | Polkadot leverages libp2p as part of its Substrate-based architecture. Libp2p enables Polkadot's scalability by facilitating seamless communication across its heterogeneous blockchain network. |
+| **[Prysm](https://prysmaticlabs.com/)** | Developed by Prysmatic Labs, Prysm is powered by libp2p powers the Ethereum consensus network alongside other clients, contributing to client diversity in the Ethereum ecosystem. Prysm of often powers [~1/3 of the Beacon Chain](https://monitoreth.io/nodes). |
 | **[Spacedrive](https://www.spacedrive.com/)** | Spacedrive implements a lazy connection system using libp2p, which establishes connections only when needed and closes them after a period of inactivity, enhancing resource efficiency. |
```

Comment:
```suggestion
| **[Aave](https://aave.com/)** | Leveraging libp2p's flexibility and robust network infrastructure, Aave has successfully expanded to multiple blockchain networks, including Ethereum, Polygon, and Avalanche. |
| **[Spacedrive](https://www.spacedrive.com/)** | Spacedrive implements a lazy connection system using libp2p, which establishes connections only when needed and closes them after a period of inactivity, enhancing resource efficiency. |
```

Reply:
adding Aave

Reply:
Do they use libp2p? I couldn't find anything in their Github which pointed to their use of libp2p apart from inheriting from Chainlink's ccip https://github.com/search?q=org%3Aaave%20libp2p&type=code